src/com/palmergames/bukkit/towny/db/TownyLegacyFlatFileConverter.java

```diff
@@ -43,32 +329,74 @@ public static void applyName(File file, String name) {
 		}
 	}
 
-	private static boolean hasName(File file) {
+	private static void applyUUIDAndName(File file, String name, String uuid) {
+		if (file.exists() && file.isFile()) {
+			try {
+				BufferedWriter bw = new BufferedWriter(new FileWriter(file, true));
+				if (!hasKey(file, "uuid"))
+					bw.append("uuid=" + uuid + "\n");
+				if (!hasKey(file, "name"))
+					bw.append("name=" + name);
+				bw.close();
+			} catch (IOException e) {
+				e.printStackTrace();
+			}
+		}
+	}
+
+	private void setKeyValueInFile(String key, String oldValue, String newValue, File file) {
+		String oldLine = key + "=" + oldValue;
+		String newLine = key + "=" + newValue;
+
+		try {
+			Scanner sc = new Scanner(file);
+			StringBuffer buffer = new StringBuffer();
```

Comment:
A plain StringBuilder can be used here instead of a buffer, StringBuilder is apparently slightly faster.